File: server/src/main/java/com/server/account/Account.java
package com.server.account;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class Account {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String name;
    private double amount;

    public Account() {

        this.name = "";
        this.amount = 0;
    }
    public Account(String name, double amount) {

        this.name = name;
        this.amount = amount;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public double getAmount() {
        return amount;
    }

    @Override
    public String toString() {
        return "Account{" +
                "name='" + name + '\'' +
                ", amount=" + amount +
                '}';
    }
}

File: client/src/app/account/account.ts
export class Account {
    id: string;
    name: string;
    amount: string;

    constructor(id: string, name: string, amount: string) {
        this.id = id;
        this.name = name;
        this.amount = amount;
    }

    //getters
    getId() {
        return this.id;
    }
    getName() {
        return this.id;
    }
    getAmount() {
        return this.id;
    }

    //setters
    setId(id:string) {
        this.id = id;
    }
    setName(name:string) {
        this.name = name;
    }
    setAmount(amount:string) {
        this.amount = amount;
    }
}
